custom_components/tesla_custom/cover.py

```diff
@@ -63,11 +63,14 @@ async def async_open_cover(self, **kwargs):
 
     @property
     def is_closed(self):
-        """Return True if charger door is open."""
-        if self._car.charge_port_latch == "Engaged":
+        """Return True if charger door is closed."""
+        if (
```

Comment:
Can't you just return the not of this test?

Reply:
For sure - I think I'm just going to return the not of `self._car.is_charge_port_door_open` to keep it simple. Not sure checking `self._car.charge_port_latch` as well is necessary unless there's something wrong with the `charge_port_door_open` key.